cmd/ocsp-updater/main_test.go

```diff
@@ -285,6 +287,57 @@ func TestMissingReceiptsTick(t *testing.T) {
 	test.AssertNotError(t, err, "Failed to run missingReceiptsTick")
 }
 
+/*
+ * Boulder Issue #1872 identified that the `getSerialsIssuedSince` function may
```

Comment:
I typically use full GitHub URLs for issues. That way they're clickable, but more importantly, in the future when we migrate to a hyper-decentralized quantum torrent based repository, there will still be a unique reference for the issue that doesn't depend on knowing GitHub conventions.